tests/rptest/tests/throughput_limits_snc_test.py

```diff
@@ -166,6 +166,16 @@ def get_config_parameter_random_value(self, prop: ConfigProp):
                 elif r == 2:
                     tc_item['client_id'] = 'client_\d+'
 
+                principals = []
+                for _ in range(self.rnd.randrange(3)):
+                    r = self.rnd.randrange(5)
```

Comment:
question: I'm guessing these additional 2 elements are for PP and SR in the future?

Reply:
exactly, great spot! Athough that doesn't hurt, better to keep it at 3 in this PR.